cabal.project

```diff
@@ -146,3 +146,21 @@ source-repository-package
     typed-protocols-examples
     network-mux
     Win32-network
+
+source-repository-package
+  type: git
+  location: https://github.com/bitemyapp/esqueleto
+  tag: 305b11e58e6a7cff9d639d409f0ace88cf154af0
+  --sha256: 0cihcscwgs0d8wmvbm9934lj6ghdyswil26dc0nhrfzdjj7iig4g
+  subdir:
+    .
+
+source-repository-package
+  type: git
+  location: https://github.com/kderme/persistent
+  tag: 2d02c002761ba0ed05ce95aeef1dfc7bd3caa073
+  --sha256: 0jl6ph2cgrl3n2f6j7ff9gqm0scz2dwxq5vs1q1ijxlli37zfzsx
```

Comment:
This is great, but I wonder if it might be more sensible to wait until these are formally released.